src/service/AliasService.js

```diff
@@ -26,3 +23,19 @@ export const deleteAlias = async(account, alias) => {
 
 	return axios.delete(url).then((resp) => resp.data)
 }
+
+class AliasService {
```

Comment:
@ChristophWurst can we use classes?

Reply:
possible but typically not required. what would be the benefit here?

Reply:
it's much nicer ;)